report60.html

```diff
@@ -131,350 +131,207 @@ <h1 style="text-align:center;">Modules PRs that require review</h1>
-  
-  
-  
-      
-        <td>74</td>
-      
-  
-  
-  
-      
-        <td>DavidS <span class='label label-primary'>iac</span> <span class='label label-warning'>puppet</span></td>
-      
-  
-  
-  
-      
-        <td>(maint) Disable acceptance testing on appveyor</td>
-      
-  
-  
-  
-      
-        <td>Rebased and marked as draft until we gain enough confidence in the new test infra to drop the old one.</td>
-      
-  
-  
-  
-      
-        <td>DavidS</td>
-      
-  
-  
-  
-      
-        <td>70</td>
-      
-  
-  
-  
-      
-        <td>1</td>
-      
-  
-  
-  
-      
-        <td>false</td>
-      
-  
-  
-  
-      
-        <td>false</td>
-      
-  
-  
-  </tr>
 
-  <tr>
-  
-  
-      <td><a target ="_blank" href='https://github.com/puppetlabs/puppetlabs-motd'>puppetlabs-motd</a></td>
-  
-  
-  
-      
-  
-  
-  
-      <td><a target ="_blank" href='https://github.com/puppetlabs/puppetlabs-motd/pull/347'>347</a></td>
-  
-  
-  
-      
-        <td>74</td>
-      
-  
-  
-  
-      
-        <td>DavidS <span class='label label-primary'>iac</span> <span class='label label-warning'>puppet</span></td>
-      
-  
-  
-  
-      
-        <td>(maint) remove platforms from release_checks that are checked on PR</td>
-      
-  
-  
-  
-      
-        <td>Rebased and marked as draft until we gain enough confidence in the new test infra to drop the old one.</td>
-      
-  
-  
-  
-      
-        <td>DavidS</td>
-      
-  
-  
-  
-      
-        <td>70</td>
-      
-  
-  
-  
-      
-        <td>1</td>
-      
-  
-  
-  
-      
-        <td>false</td>
-      
-  
-  
-  
-      
-        <td>false</td>
-      
-  
-  
-  </tr>
 
-  <tr>
-  
-  
-      <td><a target ="_blank" href='https://github.com/puppetlabs/puppetlabs-stdlib'>puppetlabs-stdlib</a></td>
-  
-  
-  
-      
-  
-  
-  
-      <td><a target ="_blank" href='https://github.com/puppetlabs/puppetlabs-stdlib/pull/1136'>1136</a></td>
-  
-  
-  
-      
-        <td>81</td>
-      
-  
-  
-  
-      
-        <td>npwalker <span class='label label-warning'>puppet</span></td>
-      
-  
-  
-  
-      
-        <td>(FEAT) Add load_csv function</td>
-      
-  
-  
-  
-      
-        <td>Hi @npwalker, I reviewed your PR. The idea is awesome, but you need to add test in spec/functions the same as loadjson_spec.rb but for load_csv function, and also to resolve some rubocop offenses. Also it will be good to rename the function to be 'loadcsv' to be like loadjson or loadyaml.
-
-kind regards,
-@adrianiurca </td>
-      
-  
-  
-  
-      
-        <td>adrianiurca</td>
-      
-  
-  
-  
-      
+
+
         <td>71</td>
-      
-  
-  
-  
-      
-        <td>2</td>
-      
-  
-  
-  
-      
+
+
+
+
+
+        <td>4</td>
+
+
+
+
+
         <td>false</td>
-      
-  
-  
-  
-      
+
+
+
+
+
         <td>true</td>
-      
-  
-  
+
+
+
   </tr>
- 
+
 </div>
 </table>
 </body>
```

Comment:
There is quite a few things removed here, is this being done by accident?

Reply:
There has also been changes made in report90.html is this also intentional?

Reply:
No, there were conflicts discovered, so I used the guidance given by @sheenaajay a few few ago and applied that knowledge to fix the conficts.

Reply:
This was no resolved, therefore please don't resolve the comment.
I don't think your supposed to be changing and committing this file. 
@daianamezdrea can you confirm for me please?

Reply:
Hello @Disha-maker , just for a better understanding:
1. Each html file is representing a report ( in our case report.html is PR's that requires review report, report30.html is the report which contains the prs with no activity for 30 days, etc), and each .html file was generated by running a ruby script + erb template ===> resulting the report page. 
2. So everytime you run bundle exec ... token ( command for running the script and erb template) a local version of the report, DIFFERENT from the one CM's GA nightly is generating everynight ( because is depending on what info do you have at the moment of the run), from here also the conflicts ( the difference between your local reports (30,60,90 days report) and the upstream ones
3. If you look in git ignore file, there should be some files added to be ignored (also the html files because are generated each time when the project is run)

Reply:
Hi, @daianamezdrea so do you think I should be making the changes suggested by @sanfrancrisko Should I leave it the way it is? Refer to the discussion after @TP comments